recipes/fmt/all/conanfile.py

```diff
@@ -50,7 +51,8 @@ def generate(self):
             tc.generate()
 
     def layout(self):
-        cmake_layout(self, src_folder="src")
+        if not self.options.header_only:
+            cmake_layout(self, src_folder="src")
```

Comment:
I do not recommend a conditional layout based on an option in layout, it's broken.

Reply:
Perfect, I was expecting this to be wonky 😆 

should I always use cmake or manually setup things?

Reply:
cmake_layout, no condition